dot/peerset/peerset.go

```diff
@@ -379,6 +382,7 @@ func (ps *PeerSet) allocSlots(setIdx int) error {
 		}
 
 		if err = peerState.tryOutgoing(setIdx, peerID); err != nil {
+			logger.Errorf("could not set peer as outgoing connection, peer: %s, error: %d", peerID.Pretty(), err)
```

Comment:
```suggestion
			logger.Errorf("could not set peer %s as outgoing connection: %s", peerID.Pretty(), err)
```

Reply:
Note the previous code contains `%d` for the error, so at least fix that part :wink: